axes/conf.py

```diff
@@ -55,3 +55,5 @@ class MyAppConf(AppConf):
     IP_WHITELIST = None
 
     IP_BLACKLIST = None
+
+    USE_LEGACY_LOGIN_VIEW_PATCH = True
```

Comment:
This should probably be just `PATCH_LOGIN_VIEW` which would translate to `AXES_PATCH_LOGIN_VIEW` if we are unsure about the deprecation at the moment.